data_pipeline/data_reading.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+import re
+import os
+import definitions
+
+def extract_triples(path: str):
+  """
+  Return a list of (id, tokens, amr string) triples for an AMR data file.
+  """
+  # Read file data into a string.
+  file = open(path,'r')
+  text = file.read()
+  file.close()
+  # Separate the data for each amr (id, token, alignemnts, amr).
+  data_blocks = text.split('\n\n')
+  triples = []
+  for data_block in data_blocks:
+    # Use regex to extract the needed info from a text block.
+    pattern = r'# ::id(.*)\n# ::tok(.*)\n# ::alignments(.*)\n(?=\()(.*)'
```

Comment:
This looks scary 😃 Maybe we can talk a little bit about this one.
Also can we extract this as a costant?

Reply:
Extracted it as a constant and put a comment to be easier understood.